### PR TITLE
Resize all columns when producing default output for global grouping sets

### PR DESCRIPTION
**Pull request by aditi-pandit**
Follow up for https://github.com/facebookincubator/velox/commit/20ac669a01482fc49ce5f281079076493a9093ad#r131312963

### CODE REVIEW

File: velox/exec/GroupingSet.cpp [hunk 1]
```diff
@@ -630,7 +630,8 @@ bool GroupingSet::getDefaultGlobalGroupingSetOutput(
   VELOX_CHECK(getGlobalAggregationOutput(iterator, globalAggregatesRow));
 
   // There is one output row for each global GroupingSet.
-  result->resize(globalGroupingSets_.size());
+  auto numGroupingSets = globalGroupingSets_.size();
```
Comment (xiaoxmeng) [inline velox/exec/GroupingSet.cpp:633]: nit: const auto numGroupingSets

**Review by xiaoxmeng — APPROVED** [review]
@aditi-pandit LGTM. Thanks!

**Review by mbasmanova — COMMENTED** [review]
@aditi-pandit Is this a refactoring or a material change? If the former, please, update PR title to clarify. If the latter, should we add a test?

**Comment by aditi-pandit** [comment]
> @aditi-pandit Is this a refactoring or a material change? If the former, please, update PR title to clarify. If the latter, should we add a test?

@mbasmanova : There are several unit tests for this code-path and I didn't encounter any problems. But the comment posted on the commit seemed valid so I went ahead with the PR. Have changed the title to reflect the change.

File: velox/exec/GroupingSet.cpp [hunk 2]
```diff
@@ -656,9 +657,10 @@ bool GroupingSet::getDefaultGlobalGroupingSetOutput(
   // aggregates.
   for (auto i = firstAggregateCol; i < outputType->size(); i++) {
     auto resultAggregateColumn = result->childAt(i);
+    resultAggregateColumn->resize(numGroupingSets);
```
Comment (mbasmanova) [inline velox/exec/GroupingSet.cpp:660]: This is the change, right? 

I believe these will no longer be needed after a change to RowVector::resize to resize children that @kgpai is working on.

**Review by mbasmanova — APPROVED** [review]
Thanks.

File: velox/exec/GroupingSet.cpp (hunk 2 above)
Comment (aditi-pandit) [reply velox/exec/GroupingSet.cpp:660]: @mbasmanova : Yes, that is the change.

The change @kgpai is doing will be great. Its very error prone to add such code for resizing all children each time.

**Pull request closed without merging**